sdk/src/types/block/core.rs

```diff
@@ -292,6 +299,7 @@ impl Block {
     /// Creates a new [`BlockBuilder`] to construct an instance of a [`BasicBlock`].
     #[inline(always)]
     pub fn build_basic(
+        protocol_params: ProtocolParameters,
         network_id: u64,
```

Comment:
Shouldn't this also be used from the ProtocolParameters now (with network_name_to_id())?

Reply:
I did not know that was possible, thanks 👍

Reply:
There is even `ProtocolParameters::network_id()` which calls this

Reply:
done